### ISSUE TITLE
How I include this into my project..? I don't able to find any dependency and .jar file ??  plz guide me.

### ISSUE THREAD

**Issue by ahmadbintariq4u** [HanSolo/charts#39]
Download link also broken

**Comment by HanSolo** [comment]
Just look over at Maven central you could either use maven:
```xml
<dependency>
  <groupId>eu.hansolo.fx</groupId>
  <artifactId>charts</artifactId>
  <version>11.7</version>
</dependency>
```
or Gradle:
```
implementation 'eu.hansolo.fx:charts:11.7'
```

**Comment by ahmadbintariq4u** [comment]
@HanSolo  
Thanks a lot. Your library is amazing. but I also am not able to see the example project. Is there is any template or example project?

**Comment by HanSolo** [comment]
For each chart there is a demo in the tests. So if you just take a look at the source of the charts library and check the test packages you should be able to figure out how to use it.

**Comment by kqtrinh** [comment]
Hi To the HanSolo Charts Developers,

I am interested experiment with your library of plots the polar and the radar plots in particular. I am quite new to Java. I tried to best to update my Java development environment to OpenJDK16 with javaFX-sdk-16. Latest Eclipse Oomph and grade 7.1.1 to try to run the radar chart test application. I messed with with on the fly when running into problems and trying to solve it by googling.

Had spent ~1 day on it without any luck to get the test application working. I am at a point where at least entire project is imported into Eclipse and the the radar test application is no longer showing a bunch of red lines. When run, I encountered below error:

WARNING: Unknown module: charts specified to --patch-module
Error: Could not find or load main class eu.hansolo.fx.charts.RadarChartTest
Caused by: java.lang.ClassNotFoundException: eu.hansolo.fx.charts.RadarChartTest

Also tried googling for a solution but I got no where so far.

Your vast supported plots are so valuable for the scientific community. I am wondering if there is any documentation to guide one to at least get pass the setting up of development environment (the basic toolchain required to run/develop with your library) so one can develop their FX app to consume your library.

Any help would be greatly appreciated in this regards,
Thanks,
